docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,508 @@
+
+-   On the menu bar, go to **IoT > IoT Setup Manager**.
+-   On the toolbar, click the IoT Setup Manager icon as shown in the following image:
+
+    ![IoT Setup Manager menu path](media/tizen_studio_ism.png)
+
+#### Launch without using tizen studio
+
+During installation, the Package Manager creates shortcuts for IoT Setup Manager, which are used to launch the IoT Setup Manager as a standalone application. To launch the IoT Setup Manager for:
+
+-   Windows, go to **Start Menu > Programs > Tizen Studio > Tools > IoT Setup Manager**.
+-   Ubuntu, go to **Dash (equivalent to Start Menu) > Applications > IoT Setup Manager**.
+
+    ![IoT Setup Manager Main](media/setup_manager_main_window.png)
+
+### Flash SD card using IoT setup manager
+
+To flash Tizen on your SD card using the IoT Setup Manager:
+
+Profiles are an easy way to store image paths and network configuration information. After you save a profile, you can use it to flash your SD Card instead of specifying all information again.
+
+1.  Create a profile:
+    1.   Click **Create** to create a profile. The **Edit Profile** window appears.
+      
+         ![IoT Setup Manager Edit](media/ism_edit_rpi4_headed.png)
+
+    2.   Provide the required information such as target, local paths to boot and platform image, and so on.
+
+         1.   **Profile name**: Enter a profile name, for example, Rpi4_Headed_Profile.
+         2.   **Target**: Select the board and image type from the drop-down list. For example, RPI4_HEADED.
+         3.   **Boot image**: Browse for the Tizen Boot image available on your hard drive. You must select proper images depending on the board. For more information, see [Downlaod Guide](#download-binaries).
+         4.   **Platform image**: Browse for the Tizen Platform image available on your hard drive. You must select proper images depending on the board. For more information, see [Downlaod Guide](#download-binaries).
+         5.   **RPM bundles**: 
+              -   Add the bundle directory to the Bundles panel.
+              -   Use this to install Wi-Fi plugin. Download using [Install Drivers](#install-drivers). Unzip and add path to 32-bit rpm or 64-bit rpm according to image to install.
+              -   Leave blank if no rpm installation required.
+         6.   **WiFi**: Enter the SSID and password to connect automatically to Wi-Fi. (Wi-Fi plugin installation is required). If Wi-Fi or ethernet is not required, you can clear the respective check box.
+         7.   **Ethernet**:
+              -   **IP**: 192.168.1.11
+              -   **Gateway**: 192.168.1.2
+              -   **Subnet**: 255.255.255.0
+              -   **Primary DNS Server**: 127.0.0.1
+
+     3. Click **OK** to save.
+
+2. Select the SD card on which you want to burn Tizen:
+      1. Insert your SD Card into your computer.
+      2. Select the appropriate SD Card from the drop-down list. For Example, `/dev/sdx` on Ubuntu and `F:` on Windows.
```

Comment:
Please change example to lower case:

https://github.com/Samsung/tizen-docs/pull/1293#discussion_r509330306:

Select the appropriate SD Card from the drop-down list. For example, `/dev/sdx` on Ubuntu and `F:` on Windows.